lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -170,14 +170,14 @@ export default function ContentSelector({
   };
 
   // file.id is a URL with a `blackboard://`, `d2l://` or `moodle://` prefix.
-  const selectFileAsURL = (file: File | Page) => selectURL(file.id);
+  const selectFileAsURL = (file: File) => selectURL(file.id);
```

Comment:
Most of the code that worked with files or pages handled them in a uniform way, so using one type to represent them makes this a little simpler.